WORKSPACE

```diff
@@ -36,8 +36,8 @@ pip_install()
 
 git_repository(
     name="org_pubref_rules_protobuf",
-    remote="https://github.com/pubref/rules_protobuf",
-    tag="v0.8.2",
+    remote="https://github.com/ghostwriternr/rules_protobuf",
```

Comment:
Why is this pointing to an unofficial repo? Can't the changes go upstream?

Reply:
I have rolled back this change for now because it is not relevant to this PR, but this will be needed again in future since these are the only protobuf rules compatible with Python, but the upstream rules don't work with generated protos. [A PR was raised](https://github.com/pubref/rules_protobuf/pull/196) to upstream to fix this that worked well for the Python rules, but wasn't merged because it wasn't compatible with golang. The unofficial fork incorporates this fix.

I will add this explicitly as a comment within the code and on the commit as well when this change has to be introduced, so we can review again.